modules/store/testing/src/mock_store.ts

```diff
@@ -6,11 +6,18 @@ import {
   INITIAL_STATE,
   ReducerManager,
   Store,
+  createSelector,
 } from '@ngrx/store';
 import { MockState } from './mock_state';
+import { MockSelector, MockSelectorWithProps } from './mock_selector';
 
 @Injectable()
 export class MockStore<T> extends Store<T> {
+  static selectors = new Map<
```

Comment:
Does this has to be `static`?

Reply:
I went back and forth on this one. If its static, it persists between tests so you don't have to override selectors each time the `MockStore` is instantiated. You'll usually do the setup in a beforeEach though 🤷‍♂️

Reply:
Makes sense 👍

Reply:
So in the hindsight, it shouldn't be `static` because it introduced https://github.com/ngrx/platform/issues/2044

I'll remove `static` and would also remove that `afterEach` hack